Configuration/Services.yaml

```diff
@@ -1,36 +1,45 @@
 services:
-   _defaults:
-      autowire: true
-      autoconfigure: true
-      public: false
-
-   T3docs\Examples\:
-      resource: '../Classes/*'
-      exclude: '../Classes/Domain/Model/*'
-
-   T3docs\Examples\LinkValidator\LinkType\ExampleLinkType:
-      tags:
-         -  name: linkvalidator.linktype
-
-   T3docs\Examples\Controller\AdminModuleController:
-      tags: ['backend.controller']
-
-   T3docs\Examples\EventListener\LinkValidator\CheckExternalLinksToLocalPagesEventListener:
-      tags:
-         - name: event.listener
-           identifier: 'txExampleCheckExternalLinksToLocalPages'
-
-   T3docs\Examples\EventListener\Core\Configuration\FlexFormParsingModifyEventListener:
-      tags:
-         - name: event.listener
-           identifier: 'form-framework/set-data-structure'
-           method: 'setDataStructure'
-         - name: event.listener
-           identifier: 'form-framework/modify-data-structure'
-           method: 'modifyDataStructure'
-         - name: event.listener
-           identifier: 'form-framework/set-data-structure-identifier'
-           method: 'setDataStructureIdentifier'
-         - name: event.listener
-           identifier: 'form-framework/modify-data-structure-identifier'
-           method: 'modifyDataStructureIdentifier'
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    public: false
+  T3docs\Examples\:
```

Comment:
I propose to preserve the empty lines between the defintions for better readability.

Reply:
I agree, it seems like make removes the whitespaces, which is not nice